### PR TITLE
Implementation channels REST endpoins

### PR DESCRIPTION
**Pull request by aldebap**
The following were added/changed/fixed:

. implementation of all commands from Channel Management as REST endpoints;

### CODE REVIEW

File: lnd/lnrpc/restrpc/restrpc.go [hunk 1]
```diff
@@ -465,39 +466,878 @@ var rpcFunctions []RpcFunc = []RpcFunc{
...
+					return getNetworkStewardVoteResp, nil
+				}
+			} else {
+				return nil, errr
+			}
+		},
+	},
+	//	service bcasttransaction
+	{
+		path: "/api/v1/BcastTransaction",
```
Comment (cjdelisle) [inline lnd/lnrpc/restrpc/restrpc.go:1320]: Hmm, capital letters in rest apis are kind of rare, I think we want to use all lower case.

File: lnd/lnrpc/restrpc/restrpc.go [hunk 2]
```diff
@@ -465,39 +466,878 @@ var rpcFunctions []RpcFunc = []RpcFunc{
...
+					return estimateFeeResp, nil
+				}
+			} else {
+				return nil, errr
+			}
+		},
+	},
+	//	service sendmany
+	{
+		path: "/api/v1/transactions/many",
```
Comment (cjdelisle) [inline lnd/lnrpc/restrpc/restrpc.go:1152]: should be /sendmany

File: lnd/lnrpc/restrpc/restrpc.go (hunk 1 above)
Comment (aldebap) [reply lnd/lnrpc/restrpc/restrpc.go:1320]: I followed Dimitris's instructions, and simply used the names mapped in [rest-annotations.yaml](https://github.com/pkt-cash/pktd/blob/develop/lnd/lnrpc/rest-annotations.yaml) file. I'm in contact with him to discuss this matter.

File: lnd/lnrpc/restrpc/restrpc.go (hunk 2 above)
Comment (aldebap) [reply lnd/lnrpc/restrpc/restrpc.go:1152]: I followed Dimitris's instructions, and simply used the names mapped in [rest-annotations.yaml](https://github.com/pkt-cash/pktd/blob/develop/lnd/lnrpc/rest-annotations.yaml) file. I'm in contact with him to discuss this matter.

**Pull request merged**